lib/hexpm/release_tasks.ex

```diff
@@ -9,7 +9,7 @@ defmodule Hexpm.ReleaseTasks do
     :ecto_sql
   ]
 
-  @repos Application.get_env(:hexpm, :ecto_repos, [])
+  @repos Application.compile_env(:hexpm, :ecto_repos, [])
```

Comment:
I always get confused as to when to use fetch, and when to use compile. Any tips for me?

Reply:
use `compile_env` at compile-time 😎 (such as exactly here!)

Reply:
to expand,

```elixir
# foo.ex

# this is compile-time
Application.compile_env(:foo, :bar)

defmodule Foo do
  # this is compile-time too
  Application.compile_env(:foo, :bar)

  def foo do
    # this is runtime
    Application.fetch_env(:foo, :bar)

    # inside unquote, we're at compile-time
    unquote(Application.compile_env(:foo, :bar))
  end

  defmacro deffoo do
    # this is compile-time too
    Application.compile_env(:foo, :bar)
    
    quote do
      # this is... it depends how we use the macro. If we use it in module body
      # we'd use compile_env here. If we use the macro inside def, we'd use fetch_env.
    end
  end
end
```

hope it makes sense and is useful!